hoomd/md/EvaluatorPairMie.h

```diff
@@ -58,14 +58,16 @@ class EvaluatorPairMie
             Scalar m2;
             Scalar m3;
             Scalar m4;
+            Scalar mie_log2_n;
+            Scalar mie_log2_m;
 
             #ifdef ENABLE_HIP
             // set CUDA memory hints
             void set_memory_hint() const {}
             #endif
 
             #ifndef __HIPCC__
-            param_type() : m1(0), m2(0), m3(0), m4(0) {}
+            param_type() : m1(0), m2(0), m3(0), m4(0), mie_log2_n(0), mie_log2_m(0) {}
```

Comment:
```suggestion
            param_type() : m1(0), m2(0), m3(0), m4(0) {}
```
With the new fast::pow refactoring, `mie_log2_n/m` are no longer used in `evalForceAndEnergy` and are not needed.